data/base-contacts.yaml

```diff
@@ -0,0 +1,3356 @@
+contacts:
```

Comment:
A seed file makes sense to store in the repo.

The convention we've been using for personal files is to store them in the lich directory (you can browse through `dependency.lic` to see that in action).